Makefile

```diff
@@ -25,3 +25,8 @@ update-prow:
 
 lint:
 	find . -name '*.sh' -print0 | xargs -0 -r shellcheck
+
+# this will build the containers and then try to use them to build themselves again, making sure we didn't break docker support
+test: maistra-builder
```

Comment:
I'm curious if this actually works, inception in two levels :)

Reply:
it does, I tested locally doing a docker in docker in docker (and now on prow)